specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcechanges.json

```diff
@@ -0,0 +1,408 @@
+        "operationId": "ResourceChanges",
+        "tags": [
+          "Changes"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "description": "the parameters for this request for changes.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceChangesRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list of changes associated with a resource over a specific time interval.",
+            "schema": {
+              "$ref": "#/definitions/ResourceChangeList"
+            }
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Basic query": {
+            "$ref": "./examples/ResourceChanges.json"
+          },
+          "First page query": {
+            "$ref": "./examples/ResourceChangesFirstPage.json"
+          },
+          "Next page query": {
+            "$ref": "./examples/ResourceChangesNextPage.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceGraph/resourceChangeDetails": {
+      "post": {
+        "description": "Get resource change details.",
+        "operationId": "ResourceChangeDetails",
+        "tags": [
+          "Changes"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "description": "The parameters for this request for resource change details.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceChangeDetailsRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource change details.",
+            "schema": {
+              "$ref": "#/definitions/ResourceChangeData"
+            }
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Basic query": {
+            "$ref": "./examples/ResourceChangeDetails.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceChangesRequestParameters": {
+      "description": "The parameters for a specific changes request.",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Specifies the resource for a changes request.",
+          "type": "string"
+        },
+        "interval": {
```

Comment:
Please add the type of 'object' to avoid linter breaking.

Reply:
I added a "type": "object" to the interval block in both specs. LintDiff still fails.